Userland/Applications/FileManager/FileUtils.h

```diff
@@ -41,10 +41,5 @@ enum class FileOperation {
 void delete_path(const String&, GUI::Window*);
 void delete_paths(const Vector<String>&, bool should_confirm, GUI::Window*);
 int delete_directory(String directory, String& file_that_caused_error);
```

Comment:
You might want to replace these with LibCore utils as well (similarly to your rm commit)

Reply:
Yeah, initially I wanted to, then I thought it to be a bit too complex, but looking back to it, I think it shouldn't be that bad.

I'd need to make `remove` return the file path that caused the error alongside the OSError I think. Will look into it in an hour or so

Reply:
just replacing delete_path with the LibCore equivalent might be enough i think, the whole delete_path and delete_directory stuff might be too customized for it

Reply:
I've rewritten delete_path and delete_paths in terms of Core::File::remove